_grab_ui.py

```diff
@@ -405,6 +405,20 @@ def __init__(self, args: Namespace, opts: Options,
         self.mark_type = NoRegion  # type: Type[Region]
         self.mode = 'normal'       # type: ModeTypeStr
         self.result = None         # type: Optional[ResultDict]
+
+        # Operating System Command (OSC); command number 52
+        # c — clipboard
+        # p — primary
+        # s — secondary
+        self.copy_to = b'c'
+        if args.copy_to:
+            if args.copy_to == 'clipboard' or args.copy_to == 'c':
+                self.copy_to = b'c'
+            if args.copy_to == 'primary' or args.copy_to == 'p':
+                self.copy_to = b'p'
+            if args.copy_to == 'secondary' or args.copy_to == 's':
+                self.copy_to = b's'
```

Comment:
This is more concisely written as:

```
self.copy_to = {'primary': b'p',
                'secondary': b's'}.get(args.copy_to, b'c')
```

and I’m doubtful about the usefulness of one-letter shortcuts. If this were a utility that the user would invoke from the command line often, I’d maybe instead provide `-1` as a shortcut for `--copy-to=primary` and `-2` for `--copy-to=secondary`; but this is a kitten that is intended to be bound to keys which is a write-once read-multiple configuration effort so I’d lean towards optimizing readability.